source/lac/tridiagonal_matrix.cc

```diff
@@ -239,7 +241,8 @@ TridiagonalMatrix<number>::compute_eigenvalues()
 
   const types::blas_int nn = n();
   types::blas_int       info;
-  stev(&N, &nn, diagonal.data(), right.data(), nullptr, &one, nullptr, &info);
+  //   stev(&N, &nn, diagonal.data(), right.data(), nullptr, &one, nullptr,
+  //   &info);
```

Comment:
@drwells @dealii/dealii This is an open TODO. We need a small internal wrapper that calls the LAPACK function `stev` for double and float, but is unimplemented otherwise

Reply:
We really need to clean up the LAPACK wrappers and find some tool that can generate these things automatically in the long run.

I am surprised that changing ConstraintMatrix resulted in breaks in this unrelated linear algebra class.